Libraries/CO.CDP.Mvc.Validation/RequiredIfAttribute.cs

```diff
@@ -38,7 +75,7 @@ public class RequiredIfContainsAttribute(string dependentProperty, string contai
         // Check if value is null or empty when condition is met
         if (conditionMet && (value == null || (value is List<string> str && str.Count == 0)))
```

Comment:
It is aligned with other attribute validators. if I change conditionMet variable in RequiredIfHasValueAttribute then it will needs to be changed in others also. 
Please suggest?

Reply:
My view on PRs is I want the code to read to me. conditionMet tells me nothing, I can see it's used as part of a condition statement but what does this condition represent? Also the comment on line 75 should be removed, it doesn't tell us anything more than the code does.

Reply:
Looking at the definition, I do not understand what's behind the condition that's supposedly met:

```csharp
 bool conditionMet = propertyValue is List<string> list && list.Contains(containsValue);
```

If any of you understands it and can suggest a better name, I'm all for improving naming and therefore understanding. This whole method could do with some better named variables, but I'd need to spend more time and focus better to suggest anything.

I agree the comment doesn't help and is redundant.

Reply:
renamed to IsDependentMatchWithTarget